server/aws-lsp-codewhisperer/src/language-server/codeWhispererServer.ts

```diff
@@ -550,10 +553,11 @@ export const CodewhispererServerFactory =
             try {
                 const qConfig = await lsp.workspace.getConfiguration(Q_CONFIGURATION_SECTION)
                 if (qConfig) {
-                    codeWhispererService.customizationArn = undefinedIfEmpty(qConfig.customization)
-                    logging.log(
-                        `Inline completion configuration updated to use ${codeWhispererService.customizationArn}`
-                    )
+                    const customizationValue = undefinedIfEmpty(qConfig.customization)
+                    codeWhispererService.customizationArn = customizationValue
+                    sessionManager.getCurrentSession()?.setCustomizationArn(customizationValue)
```

Comment:
Given sessions are very short lived I was not able to trigger such a scenario in my testing. After some testing I will actually remove this line so that we get set customization arn on the sessions at creation time only